### PR TITLE
Various Additions way too late for Native American Heritage month

### LINKED ISSUE #61663: wild rice as a harvestable plant
### Is your feature request related to a problem? Please describe.

a fair number of cooking recipes requires rice specifically to be made. the problem with this is that the only real source of rice is dried rice which will eventually rot in the long run making rice effectively unobtainable.

### Solution you would like.

wild rice to be added to the game to give a way to farm or forage rice in the long run as a substitute for dried rice (that and wild rice could be dried into dried rice). certain varieties of wild rice actually do grow in the area where the game takes place so it would be a sensible addition from a realism perspective. https://en.wikipedia.org/wiki/Wild_rice

### Describe alternatives you have considered.

remove dried rice's shelf life so it won't rot away in the long run and thus become unscavengable.

### Additional context

wild rice seems to grow in similar conditions to lotuses and and cattails so it could be added to those locations. wild rice can be grown deliberately so it would make sense fore it to be plantable.

### PR DESCRIPTION
**Pull request by merrygin**
#### Summary
Content "Various additions and tweaks relating to Native American Heritage"

#### Purpose of change
November is Native American Heritage month, as some of you might know. That might be a nice opportunity to make additions that relate to Native American heritage and are beneficial to the game or should be represented in the game. For example, this would Fix #61663 by adding harvestable wild rice and make it an ingredient for several recipes.

#### Describe the solution
This is a list of things included or planned. You can refer to the numbers if you have specific comments/requests.

- [x] 1. Add Wild Rice (aka "Indian Wild Rice"), a staple of many Great Lakes and New England tribes in the past and still part of many tribes traditional cuisine. Also, native though increasingly rare plant in this area.
- [x] 2. Added dishes: Succotash, Frybread, "Indian Tacos", Hazelnut cakes, Three sisters stew, pemmican (only description change to highligh origin, also made available through recipe book, see below)
- [x] 3. Added a recipe book with Native American recipes.
- [postponed] 4. Added a simple variant of a maple spigot, which can be made by bark, as was traditionally done by tribes in New England, who invented that method.
- [x]  5.Changed Moccasin description and recipe (to make it craftable with leather and fabric cutting 1 instead of pelts and fab_cut 2)
- [x] 6. Make innawood starting scenario starting you of with historical native american gear and three sisters seeds.
- [x] 7. Add butternuts (different walnut species)? Or other species of plants native to new england that could have use to the player.
- [postponed] 8. Add some names (current or historical ones) of tribes of the area into the mix.
- [x] 9. Add squashes as seperate harvestable/plantable item, also with uses like making bottles out of it (tornikeo gave his consent to use #61007)
- [postponed] 10. more food recipes?

#### Describe alternatives you've considered
Different additions relating to Native American Heritage in New England that is beneficial to/should be represented in game? 

#### Testing
1) tested, seems to spawn and work fine now
2) all appear sensible, some remaing kcal+density issues upsetting tests, though
3) seems to work fine, spawns in shops not tested
_4) more involved solution needed than only new item, needs more time so Ill postpone this_
5) appears to work
6) appears to work
7) does not **not** work, but haven't seen one in the wild yet?! Very few/no walnut trees in general
_8) postponed_
9) all crafts work as intended, only thing not tested is if the gourd grows properly 

#### Additional context
1a) wild rice is still present in NE, although often endangered due to habitat loss https://gobotany.nativeplanttrust.org/species/zizania/palustris/ 
1b) wild rice is culturally most heavily associated with the Great Lakes region, particularly Ojibwe, unfortunately didnt find scientific reference for use in NE, but I) mentions here https://en.wikipedia.org/wiki/Southern_New_England_Algonquian_cuisine II) NE was heaviliy influenced by Eastern Agricultural Complex & Great Lakes region (many tribes in both regions are algonquian), so its probable some at least used to use wild rice. In any case, it seems to be an important food in the revival of traditional native cuisines.
2) [succotash](https://newengland.com/yankee-magazine/food/succotash-recipe-with-a-history/ ), [frybread & "Indian Tacos"](https://www.mentalfloss.com/article/637618/fry-bread-controversial-history), [hazelnut cake & 3-sisters stew](https://indigenousnh.com/2018/03/02/indigenous-foods-and-recipes/), [pemmican](https://en.wikipedia.org/wiki/Pemmican#History)
3) inspired by the[ Sioux Chef's Indigenous Kitchen](https://sioux-chef.com/) - is the naming a problem?
4) decription of traditional maple syrup production @ [abenaki and wabenaki tribes](https://abenakitribe.org/maple-syrup).
5) Mocassin history and relatively simple production (at least the base shoe, not counting the variations and decorations) [here](https://archaeologymuseum.ca/origin-moccasins/) and [here](https://www.heddels.com/2018/02/the-history-of-the-moccasin-2/).
7) butternut use [by natives and colonisers alike](https://archive.org/details/fieldguidetoedib00angi/page/48/mode/2up) still [found in NE](https://gobotany.nativeplanttrust.org/species/juglans/cinerea/).
9) See resources given in #61007.

This is a mixed bag of stuff that came to my head while researching, but I am not Native American, not even American, so please take my additions with that caveat and criticise where necessary!

Apart from critiquing my ideas, please also feel free to suggest stuff that would fit this PR!

### CODE REVIEW

**Comment by tornikeo** [comment]
Yeah, #61007 is on halt right now. Don't see myself workin on it anytime soon.

**Comment by MNG-cataclysm** [comment]
You need to lint your JSON using the official [styler](https://dev.narc.ro/cataclysm/format.html).

**Comment by merrygin** [comment]
Thanks! ^^
yeah sorry, I know, its still wip and I usually lint only at the end. should have indicated that in the title (did now).

**Comment by merrygin** [comment]
> Yeah, #61007 is on halt right now. Don't see myself workin on it anytime soon.

I see! If you don't mind, I could try and incorporate your PR here and try and finish it, because I liked your idea very much and it would be very fitting. Ofc only if thats ok with you. What do you think, @tornikeo ?

**Comment by tornikeo** [comment]
Sure, @merrygin that'd be amazing. :heart: Is there anything I can to help you with that?

**Comment by fairyarmadillo** [comment]
If you are looking for recipe ideas, hows about akutaq? It wasn't made by indigenous New Englanders afaik, but it is still very relevant to the player's situation and is a common enough recipe to find in real life books about traditional foodways or homesteading:

https://en.wikipedia.org/wiki/Alaskan_ice_cream

Tallow or lard plus fruit of any kind is probably sufficient for the recipe. The wikipedia article mentions that some people put meat in it but most of the videos I have seen show just crisco and berries. It could be a more enjoyable way for survivors to bulk up than just eating the lard by itself like everyone does.

File: data/json/items/comestibles/baked.json [hunk 1]
```diff
@@ -18,5 +18,24 @@
...
+    "type": "COMESTIBLE",
+    "id": "hazelnut_cake",
+    "name": { "str_sp": "hazelnut cake" },
+    "weight": "30 g",
+    "color": "brown",
+    "spoils_in": "7 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 160,
+    "description": "This traditional hazelnut cake (or Bag8nizab8nak cake in Abenaki) is a nice, protein-rich treat for cold days.",
```
Comment (TheShadowFerret) [inline data/json/items/comestibles/baked.json:32]: image

I don't think those accented letters worked 🤔

**Comment by merrygin** [comment]
> Sure, @merrygin that'd be amazing. ❤️ Is there anything I can to help you with that?

@tornikeo oof, ok, sorry for the silence, got sidetracked by responsibilities. I don't think I need anything apart from your PR, so I got everything already! Thanks!

File: data/json/items/comestibles/baked.json (hunk 1 above)
Comment (merrygin) [reply data/json/items/comestibles/baked.json:32]: Yeah, that actually was kind of intentional. Apparently, there is an extra letter in Algonquian languages that looks almost like an 8 and a recipe source I used actually just put an 8 where the letter should be. This one ȣ. So I did the same, assuming at first the letter is actually just an 8... Not sure if cataclysm can depict special characters like that though?! I'll try it out.

Comment (merrygin) [reply data/json/items/comestibles/baked.json]: seems to work now with the special characters!

**Review by MNG-cataclysm — APPROVED** [review]

File: data/json/items/chemicals_and_resources.json [hunk 2]
```diff
@@ -1273,6 +1273,21 @@
...
   },
+  {
+    "type": "GENERIC",
+    "id": "butternut",
+    "category": "food",
+    "symbol": "%",
+    "color": "yellow",
+    "name": { "str_sp": "butternuts" },
+    "description": "A handful of nuts from a butternut tree, still in their shell.",
+    "//spoils_in": "180 days",
```
[the remark this answers is not in the document]
Comment (merrygin) [reply data/json/items/chemicals_and_resources.json]: Ah, thanks for spotting that!

File: data/json/items/containers.json [hunk 3]
```diff
@@ -3356,6 +3356,44 @@
...
+          "material": [ { "type": "wood", "covered_by_mat": 100, "thickness": 2.0 } ],
+          "encumbrance": 3,
+          "coverage": 8,
+          "covers": [ "torso" ],
+          "specifically_covers": [ "torso_hanging_front" ]
+        }
+      ]
+    },
+    "flags": [ "BELTED", "OVERSIZE", "WATER_FRIENDLY" ]
+  },
```
Comment (bombasticSlacks) [inline data/json/items/containers.json:3396]:
```suggestion
  }
  {
```

sorry I messed up the merge

**Pull request merged**